modules/utils/info_util.py

```diff
@@ -240,7 +242,7 @@ def send_post_to_admins(self) -> bool:
         message = self.__message.reply_to_message
         group_id = Config.meme_get('group_id')
         poll = message.poll  # if the message is a poll, get its reference
-
+        
```

Comment:
I suggest you whitespace plugin for vscode to remove automatically useless spaces